docs/release-notes/8.0.35-27.md

```diff
@@ -0,0 +1,69 @@
+# Percona XtraDB Cluster 8.0.35-27 (2024-01-16)
+
+[Get started with Quickstart Guide for Percona XtraDB Cluster.](../quickstart-overview.md)
+
+Percona XtraDB Cluster supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+## Release highlights
+
+Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.0.35-27 (2023-12-27) release notes](https://docs.percona.com/percona-server/8.0/release-notes/8.0.35-27.html).
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.0.35 and included in Percona XtraDB Cluster are the following:
+
+* Upgraded the linked OpenSSL library to OpenSSL 3.0.10.
+* Removed the printed query string limit to display the characters for a detected deadlock section of the engine status log.
+* Fixed a procession of single-character tokens by an FTS parser plugin.
+
+### Deprecations
+
+A future release may remove deprecated variables and options. The usage of these deprecated items may cause a warning. We recommend migrating from deprecated variables and options as soon as possible.
+
+This release deprecates the following variables and options:
+
+* The `binlog_transaction_dependency_tracking` server system variable
+
+* The `old` and `new` server system variables
+
+* The `--character-set-client-handshake` server variable
+
+* `INFORMATION_SCHEMA.PROCESSLIST` and the `SHOW PROCESSLIST` command
+
+* The `performance_schema_show_processlist` variable
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.35 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-35.html).
+
+## Bug fixes
+
+* [PXC-4343](https://perconadev.atlassian.net/browse/PXC-4343): The table spaces were corrupted during SST that caused the Xtrabackup failure with the `Header page contains inconsistent data in datafile` error.
```

Comment:
Maybe it worth mentioning special credits to Andrew Garner, who provided information necessary to fix the issue?

Reply:
Done.